pytest.ini

```diff
@@ -3,7 +3,7 @@
 addopts = --durations=30 -ra
 testpaths =
   lib/ramble/ramble/test
-  var/ramble/repos/*/*/*/test
+  #var/ramble/repos/*/*/*/test/*.py
```

Comment:
Just double checking, is this intended?

Reply:
I don't know. You changed it in this commit: https://github.com/GoogleCloudPlatform/ramble/commit/7f8ab379ff063abf783b04f2d86531e333a53738 
Do you remember what it was about?

Reply:
Reverted